Makefile

```
@@ -79,6 +79,9 @@ OPERATOR_NAMESPACE = mongodb-atlas-system
 ATLAS_DOMAIN = https://cloud-qa.mongodb.com/
 ATLAS_KEY_SECRET_NAME = mongodb-atlas-operator-api-key
 
+GO_LICENSES = go-licenses
+BAD_LICENSES = restricted,reciprocal
```

[nit] they're not bad they are just not ideal
```suggestion
DISALLOWED_LICENSES = restricted,reciprocal
```

---

Right!

fixing